class-sc-wp-updates-notifier.php

```diff
@@ -391,21 +387,17 @@ private function plugins_update_check( $all_or_active ) {
 		/**
 		 * Check to see if any theme updates.
 		 *
-		 * @param int $all_or_active Should we look for all themes or just active ones.
-		 *
 		 * @return bool
 		 */
-		private function themes_update_check( $all_or_active ) {
+		private function themes_update_check() {
 			$settings = $this->get_set_options( self::OPT_FIELD ); // get settings
 			do_action( 'wp_update_themes' ); // force WP to check for theme updates
 			$update_themes = get_site_transient( 'update_themes' ); // get information of updates
 			$theme_updates = array(); // array to store all the theme updates
 			if ( ! empty( $update_themes->response ) ) { // any theme updates available?
 				$themes_need_update = $update_themes->response; // themes that need updating
-				if ( 2 === $all_or_active ) { // are we to check just active themes?
-					$active_theme       = array( get_option( 'template' ) => array() ); // find current theme that is active
-					$themes_need_update = array_intersect_key( $themes_need_update, $active_theme ); // only keep theme that is active
-				}
+				$active_theme       = array( get_option( 'template' ) => array() ); // find current theme that is active
```

Comment:
Little bit of validation,

```suggestion
				$active_theme       = array( ( (string) get_option( 'template' ) ) => array() ); // find current theme that is active
```